packages/central-server/src/apiV2/utilities/constructNewRecordValidationRules.js

```diff
@@ -79,7 +79,7 @@ export const constructForSingle = (models, recordType) => {
           // Always require a permission group ID, by default an item should at least be public.
           constructRecordExistsWithId(models.permissionGroup),
         ],
-        type: [constructIsOneOf(FEED_ITEM_TYPES)],
+        type: [isAString],
```

Comment:
Losing a bit of validation here... so we could change this. There wasn't exactly a clear pattern for exporting consts from the model classes, so I didn't do it here, but perhaps we just set the const as a static property of the model class?